dtmi/stmicroelectronics/steval_proteus1/standard_if-1.json

```diff
@@ -0,0 +1,301 @@
+        ]
+      }
+    },
+    {
+      "@type": "Telemetry",
+      "comment": "Acceleration Peak Status X/Y/Z components [m/sˆ2]",
+      "displayName": {
+        "en": "Acc Peak Status [m/sˆ2]"
+      },
+      "name": "predictiveaccelerationstatus",
+      "schema": {
+        "@type": "Object",
+        "description": {
+          "en": "Sample format I.e.: {statusacc_x, statusacc_y, statusacc_z, accpeak_x, accpeak_y, accpeak_z}."
+        },
+        "displayName": {
+          "en": "Single Predictive Acceleration Peak Status sample."
+        },
+        "fields": [
+          {
+            "displayName": {
+              "en": "[AccPeak] Status Acceleration X:"
+            },
+            "name": "statusacc_x",
+            "schema": "double"
+          },
+          {
+            "displayName": {
+              "en": "[AccPeak] Status Acceleration Y:"
+            },
+            "name": "statusacc_y",
+            "schema": "double"
+          },
+          {
+            "displayName": {
+              "en": "[AccPeak] Status Acceleration Z:"
+            },
+            "name": "statusacc_z",
+            "schema": "double"
+          },
+          {
+            "displayName": {
+              "en": "Acceleration Peak X:"
+            },
+            "name": "accpeak_x",
+            "schema": "double"
+          },
+          {
+            "displayName": {
+              "en": "Acceleration Peak Y:"
+            },
+            "name": "accpeak_y",
+            "schema": "double"
+          },
+          {
+            "displayName": {
+              "en": "Acceleration Peak Z:"
+            },
+            "name": "accpeak_z",
+            "schema": "double"
+          }
+        ]
+      }
+    },
+    {
+      "@type": "Telemetry",
+      "comment": "Speed Status X/Y/Z components [mm/s]",
+      "displayName": {
+        "en": "Speed Status  [m/sˆ2]"
+      },
+      "name": "predictivespeedstatus",
+      "schema": {
+        "@type": "Object",
+        "description": {
+          "en": "Sample format I.e.: {statusacc_x, statusacc_y, statusacc_z, rmsspeed_x, rmsspeed_y, rmsspeed_z}."
+        },
+        "displayName": {
+          "en": "Single Predictive Speed Status sample."
+        },
+        "fields": [
+          {
+            "displayName": {
+              "en": "[Speed] Status Acceleration X:"
+            },
+            "name": "statusacc_x",
+            "schema": "double"
+          },
+          {
+            "displayName": {
+              "en": "[Speed] Status Acceleration Y:"
+            },
+            "name": "statusacc_y",
+            "schema": "double"
+          },
+          {
+            "displayName": {
+              "en": "[Speed] Status Acceleration Z:"
+            },
+            "name": "statusacc_z",
+            "schema": "double"
+          },
+          {
+            "displayName": {
+              "en": "RMS SPEED X:"
+            },
+            "name": "rmsspeed_x",
```

Comment:
perhaps capital S, here and below: "rmsSpeed"

Reply:
[335ab63](https://github.com/Azure/iot-plugandplay-models/pull/449/commits/335ab63428b102711a254df3e3b7f64320e9601a)